### DISCUSSION TITLE
Option to Always Reveal Action List in Search App

### DISCUSSION THREAD

**Discussion by CompuGeniusCode** [ramensoftware/windhawk-mods#1187] [General]
Is it possible to add an option that will automatically show the Action List here:
![image](https://github.com/user-attachments/assets/6b1bc639-b5d9-49d0-9a75-1bedcb2a9c77)

**Comment by m417z** [comment]
Currently, there's no such mod. In fact, no Windhawk mods exist so far that customize the search UI, which is implemented as a web view control.
